samples/cloud-client/snippets/authenticate_with_api_key.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START auth_cloud_authenticate_api_key]
+
+from google.cloud import language_v1
+
+
+def authenticate_with_api_key(quota_project_id: str, api_key_string: str) -> None:
+    """
+    Demonstrate authenticating with an API key for Google Language service.
+
+    // TODO(Developer):
+    //  1. Make sure you have the necessary permission to use Google Language service.
+
+    Args:
+        quota_project_id: Google Cloud project id that should be used for quota and billing purposes.
+        api_key_string: The API key to authenticate to the service.
+    """
+
+    # Initialize the Language Service client and set the API key and the quota project id.
+    client = language_v1.LanguageServiceClient(client_options={"api_key": api_key_string,
+                                                               "quota_project_id": quota_project_id})
```

Comment:
This is interesting-- there's no way to create a client with an API key and no quota project? I thought that was the point of the API key, that it provided the project to be billed...

Reply:
I've followed up with this in an internal bug. Will update it here post response from the team.